emr-dynamodb-hive/src/main/java/org/apache/hadoop/hive/dynamodb/DynamoDBSerDe.java

```diff
@@ -90,6 +92,11 @@ public ObjectInspector getObjectInspector() throws SerDeException {
     return Text.class;
   }
 
+
+  protected HiveDynamoDBType getTypeObjectFromHiveType(String type) {
```

Comment:
What's the point in extracting this as a method?

Reply:
It's so we can use the [ListTypeFactor here](https://github.com/awslabs/emr-dynamodb-connector/pull/71/files#diff-01feb7d63da4958c25fa3fb6ba334509R9).

Reply:
ah makes sense